tests/integration/test_vulnerability_detector/conftest.py

```diff
@@ -2,6 +2,8 @@
 # Created by Wazuh, Inc. <info@wazuh.com>.
 # This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
 
+import os
+import time
```

Comment:
These modules are not used.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/721/commits/a162404e20a560e8e14b7e1aa683f5b4d3d67c3e